command/build.go

```diff
@@ -23,12 +23,13 @@ type BuildCommand struct {
 }
 
 func (c *BuildCommand) Run(args []string) int {
-	var cfgColor, cfgDebug, cfgForce, cfgTimestamp, cfgParallel bool
+	var cfgColor, cfgDebug, cfgConnInfo, cfgForce, cfgTimestamp, cfgParallel bool
 	var cfgOnError string
 	flags := c.Meta.FlagSet("build", FlagSetBuildFilter|FlagSetVars)
 	flags.Usage = func() { c.Ui.Say(c.Help()) }
 	flags.BoolVar(&cfgColor, "color", true, "")
 	flags.BoolVar(&cfgDebug, "debug", false, "")
+	flags.BoolVar(&cfgConnInfo, "connection-info", false, "")
```

Comment:
I think `communication-credentials` includes all the things this does, it also prints IPs ports etc. Maybe something like `--communication-details`? It be awesome if this was a bit shorter since this will probably most used by humans troubleshooting and developing builds.

Reply:
I agree 🙂  may be: `comm-credentials` ?